src/main/java/ja/workshops/hibernate/parts/entity_manager_persistence_xml/README.adoc

```diff
@@ -0,0 +1,54 @@
+# Entity Manager
```

Comment:
Do przeróbki na ADOCową, kod oznaczyć, czy to katalogi czy wywołania czy nazwy klas.